tasks/main.yml

```diff
@@ -57,10 +57,10 @@
     dest: "{{ composer_home_path }}/auth.json"
     owner: "{{ composer_home_owner }}"
     group: "{{ composer_home_group }}"
-  when: composer_github_oauth_token
+  when: composer_github_oauth_token | bool
```

Comment:
This is wrong. This is not a boolean and now it forces oauth token to always be skipped.

Reply:
Opened PR #60 to correct this